README.md

```diff
@@ -341,6 +341,7 @@ Services to securely store your Docker images.
 * [Helios](https://github.com/spotify/helios) - A simple platform for deploying and managing containers across an entire fleet of servers by [@spotify](spotify)
 * [Kontena](https://github.com/kontena/kontena) - Application Containers for Masses https://www.kontena.io/
 * [Kubernetes](https://github.com/kubernetes/kubernetes) - Open source orchestration system for Docker containers by Google
+* [Lumogon](https://github.com/puppetlabs/lumogon) - Inspect, report and analyze container applications by Puppet
```

Comment:
Awesome! But I kindly ask you to make a new PR for this tool as it has nothing to do with the current PR. As commented earlier, please come back again later with a new PR.